user-guide/Getting_started/DaaS/Removing_a_DaaS_system.md

```diff
@@ -5,9 +5,11 @@ uid: Removing_a_DaaS_system
 # Removing a DaaS system
 
 > [!WARNING]
-> When you remove a DaaS system, this is **irreversible**, and **all data of the system will be lost**.
+> When you remove a DaaS system, the system will be decommissioned. Within 7 days after you remove the system, you can still recover it by contacting [daas@dataminer.services](mailto:daas@dataminer.services). However, the cloud identity and shares will be lost, and the system will receive a new cloud identity within your organization upon recovery.
 
-1. On the dataminer.services page, click the ellipsis ( "...") button in the top-right corner of the box representing the system and select *Delete DMS*.
+1. Make sure you are logged in to dataminer.services with an account that has the necessary permissions for the DaaS system in your organization on dataminer.services.
```

Comment:
It might be more helpful here if we detailed exactly which permissions are needed. Is this something we can add?

Reply:
Might be good yes, currently you'd need the admin or owner role.

Reply:
Might be good yes :)

Currently you'd need the **Admin or Owner role on organization level** or **Owner role on system level** to delete a DaaS system.